.github/workflows/test-os.yml

```diff
@@ -229,17 +229,14 @@ jobs:
           vagrant ssh -- "sudo cat /vagrant/.tmp/logs/containerd"
 
   sandbox-build:
-    runs-on: ubuntu-24.04
+    runs-on: ${{ matrix.platform == 'linux/arm64' && 'ubuntu-22.04-arm' || 'ubuntu-24.04' }}
```

Comment:
Why 22.04?

Reply:
This is related to https://github.com/actions/partner-runner-images/issues/28 but seems it has been fixed so will update to 24.04